aiounifi/controller.py

```diff
@@ -130,7 +128,11 @@ async def login(self) -> None:
         }
 
         await self._request("post", url=url, json=auth)
-
+        if (
+            response := self.last_response
+        ) is not None and response.status == HTTPStatus.OK:
+            if (csrf_token := response.headers.get("x-csrf-token")) is not None:
+                self.headers = {"x-csrf-token": csrf_token}
```

Comment:
```suggestion
        if (
            (response := self.last_response) is not None
            and response.status == HTTPStatus.OK
            and (csrf_token := response.headers.get("x-csrf-token")) is not None
        ):
            self.headers = {"x-csrf-token": csrf_token}
```

No need to nest if-statement